### PR TITLE
Checkers on integration-cli/docker_api_info_test

### PR DESCRIPTION
**Pull request by aditirajagopal**
Applying #16756 to integration-cli/docker_api_info_test.go

Signed-off-by: Aditi Rajagopal arajagopal@us.ibm.com

### CODE REVIEW

File: integration-cli/docker_api_info_test.go [hunk 1]
```diff
@@ -1,18 +1,18 @@
 package main
 
 import (
+	"github.com/docker/docker/pkg/integration/checker"
+	"github.com/go-check/check"
```
Comment (runcom) [inline integration-cli/docker_api_info_test.go]: gofmt will put this under stdlib imports tho :P

**Comment by runcom** [comment]
ping @vdemeester LGTM

File: integration-cli/docker_api_info_test.go [hunk 2]
```diff
@@ -32,8 +32,6 @@ func (s *DockerSuite) TestInfoApi(c *check.C) {
 
 	out := string(body)
 	for _, linePrefix := range stringsToCheck {
-		if !strings.Contains(out, linePrefix) {
-			c.Errorf("couldn't find string %v in output", linePrefix)
-		}
+		c.Assert(out, checker.Contains, linePrefix, check.Commentf("couldn't find string %v in output", linePrefix))
```
Comment (runcom) [inline integration-cli/docker_api_info_test.go]: I also think Commentf isn't needed at all here

File: integration-cli/docker_api_info_test.go (hunk 1 above)
Comment (vdemeester) [inline integration-cli/docker_api_info_test.go]:
```go
import (
	"net/http"
	"strings"

	"github.com/docker/docker/pkg/integration/checker"
	"github.com/go-check/check"
)
```

File: integration-cli/docker_api_info_test.go (hunk 2 above)
Comment (vdemeester) [inline integration-cli/docker_api_info_test.go]: Yeah, this comment is not needed :wink:

**Comment by vdemeester** [comment]
@aditirajagopal compilation error : `./docker_api_info_test.go:7: imported and not used: "strings"`

File: integration-cli/docker_api_info_test.go (hunk 1 above)
Comment (runcom) [inline integration-cli/docker_api_info_test.go]: Exactly that :p

**Comment by aditirajagopal** [comment]
ping @vdemeester

**Comment by calavera** [comment]
LGTM

**Pull request merged**